### DISCUSSION TITLE
AWESOME thank you thank you

### DISCUSSION THREAD

**Discussion by 34r7h** [Python-Cardano/pycardano#10] [General]
Although I primarily use JS in our stack, I'm going to give your work a try. Been super frustrated with car-ser-lib, especially how it doesn't handle the newer coinsperutxoword change. Similarly frustrated with running a sync'd node just to mint tokens and sign txs.

Looking forward and thank you. If I'm able to work with it, then I'll see about wrapping it in JS via https://www.npmjs.com/package/python-bridge

**Comment by cffls** [comment]
Thank you for your kind note @34r7h ! Happy to see this library to be even used in JS. Please keep me posted how it goes. If you don't mind, I will kindly convert this issue to a discussion.
